CHANGELOG.md

```diff
@@ -66,6 +73,11 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/) and this p
 -   `<TagList />`
     -   vertical alignment fixed in nowrap containers and for tags with icons
 
+### Fixed
+
+-   `<BreadcrumbsList />`
+    -   `onItemClick` handler is only executed if breadcrumb has `href` set because this is one callback parameter
```

Comment:
This entry is not in the 'Unreleased' section. Also it is exactly clear to me what the reason means.